lib/twitter_cldr/js/mustache/bundle.coffee

```diff
@@ -30,6 +30,12 @@ TwitterCldr.locale = "{{current_locale}}";
 {{> utilities}}
 {{{contents}}}
 
+TwitterCldr.set_data = (bundle) ->
+  for key, sub_bundle of bundle
+    for bundle_key, bundle_value of sub_bundle
+      @[key][bundle_key] = bundle_value
```

Comment:
So, when we say `@[key]` here we basically get access to one of our classes and set the value directly on that class, right? While it's kinda nice that we have data injected into every class that needs it, I feel like it makes the setup a bit more complicated than it needs to be. 

Can we save the whole bundle as it is in something like `TwitterCldr.data` and then let each class access it from there in a corresponding class method? That way we don't need to copy things over and we don't need default `{}` in every class.

Also, we need our new setup to work in a browser where you simply load implementation and data files separately (maybe not even in that order). I might have missed something, but it seems like current solution won't work if you just load both files using `script` tags in an HTML page without `require` or any other fancy package manager. For that to work we probably need data files to inject themselves straight into global `TwitterCldr` object (maybe into the same `TwitterCldr.data` property) and, possibly, define this object first if it's not defined yet. At the same time implementation bundle shouldn't override the whole `TwitterCldr` object if it's already defined. That way you can load data and implementation in any order you want.